### PR TITLE
Load the Runtime reporter test case summary from JSON file

### PR DESCRIPTION
**Pull request by sundaramrajendran**

### CODE REVIEW

File: client/src/main/resources/templates/js/main.js [hunk 1]
```diff
@@ -195,19 +195,20 @@ function displaySource(src, title) {
...
-      } else if (item.status === 'Skipped') {
-        skipped = skipped + 1;
-      } else {
-        running = running + 1;
-      }
-    });
+    if(reports.reportSummary.testMethodsSummary.passed) {
+      passed = reports.reportSummary.testMethodsSummary.passed;
+    }
+    if(reports.reportSummary.testMethodsSummary.failed) {
```
Comment (sebady) [inline client/src/main/resources/templates/js/main.js]: Would keeping the "else if" 's be preferable here.

Comment (sundaramrajendran) [inline client/src/main/resources/templates/js/main.js]: @sebady We can't, Because each if condition is not related each other

**Comment by mach6** [comment]
@sundaramrajendran @sebady looks good to me

**Comment by sebady** [comment]
merged

**Pull request closed without merging**